orchagent/neighorch.cpp

```diff
@@ -579,5 +650,438 @@ bool NeighOrch::removeNeighbor(const NeighborEntry &neighborEntry)
 
     removeNextHop(ip_address, alias);
 
+    if(gMySwitchType == "voq")
+    {
+        //Sync the neighbor to delete from the GLOBAL_APP_DB
+        voqSyncDelNeigh(alias, ip_address);
+    }
+
+    return true;
+}
+
+void NeighOrch::doVoqSystemNeighTask(Consumer &consumer)
+{
+    SWSS_LOG_ENTER();
+
+    //Local inband port as the outgoing interface of the static neighbor and static route
+    Port port;
+    if(!gPortsOrch->getInbandPort(port))
+    {
+        //Inband port is not ready yet.
+        return;
+    }
+    string nbr_odev = port.m_alias;
+    MacAddress inband_mac = gMacAddress;
+
+    auto it = consumer.m_toSync.begin();
+    while (it != consumer.m_toSync.end())
+    {
+        KeyOpFieldsValuesTuple t = it->second;
+        string key = kfvKey(t);
+        string op = kfvOp(t);
+
+        size_t found = key.find(':');
```

Comment:
Instead of hardcoded ":", can we use getTableNameSeparator()?

Reply:
Yes. We'll use getTableNameSeparator() here.